packages/node-opcua-end2end-test/test/end_to_end/test_e2e_1170.ts

```diff
@@ -0,0 +1,79 @@
+const async = require("async");
+const should = require("should");
+const opcua = require("node-opcua");
+const OPCUAClient = opcua.OPCUAClient;
+const UserTokenPolicy = opcua.UserTokenPolicy;
+const { perform_operation_on_client_session } = require("../../test_helpers/perform_operation_on_client_session");
+// require("node-opcua-service-session").UserNameIdentityToken;
+
+const describe = require("node-opcua-leak-detector").describeWithLeakDetector;
+describe("Testing bug #1170", function() {
+
+    const port = 1170;
+    let server = null;
+    let endpointUrl = "";
+
+
+    before(async () => {
+
+
+        server = new opcua.OPCUAServer({
```

Comment:
I would provide an explicit productUri in the OPCServer constructor, to make sure we verify against this one and not against the default value.

Reply:
I added the parameter